README.md

```diff
@@ -62,6 +62,15 @@ $ openstack tcib container image build --config-file containers.yaml --repo-dir
 
 The `--tcib-extras tcib_package=` is required so that non-template files are sourced from the local `container-images/kolla` instead of from the `python-tcib-containers` package in the container build.
 
+## Matching container images to tcib commits
+
+1. Look at the container image in quay.io
+   (e.g. https://quay.io/repository/podified-antelope-centos9/openstack-tempest-extras?tab=tags) and take the tag
```

Comment:
take the latest tag as any other tag would not match the current-podified

Reply:
It works for older quay.io tags too.
1. E.g. quay.io tempest-extras tag 1401eccf4b1600f1bdacc701a3fe1d9e
2. selects https://trunk.rdoproject.org/centos9-antelope/current-podified/14/01/1401eccf4b1600f1bdacc701a3fe1d9e/delorean.repo
3. points to baseurl=https://trunk.rdoproject.org/centos9-antelope/component/podified/4a/ef/4aef81881de941f0251aa848515243f666b18672_dbfdef11 which is  https://github.com/openstack-k8s-operators/tcib/commit/4aef81881de941f0251aa848515243f666b18672

Reply:
Thanks for rechecking, i misread that earlier to use latest current-podified but pointing to the hash should work fine.